src/main/java/core/basesyntax/strategy/impl/BalanceOperation.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax.strategy.impl;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.strategy.OperationHandler;
+import java.util.Map;
+
+public class BalanceOperation implements OperationHandler {
+    @Override
+    public void apply(FruitTransaction transaction, Map<String, Integer> storage) {
+        storage.put(transaction.getFruit(), transaction.getQuantity());
+    }

```

Comment:
```suggestion
    public void apply(FruitTransaction transaction) {
        Storage.storage.put(transaction.getFruit(), transaction.getQuantity());
    }
```
Fix it for other handlers